src/pybind/mgr/volumes/fs/fs_util.py

```diff
@@ -58,16 +58,17 @@ def volume_exists(mgr, fs_name):
             return True
     return False
 
-def listdir(fs, dirpath):
+def listdir(fs, dirpath, filter_entries=[]):
```

Comment:
As a best practice, you generally want to avoid a mutable default argument: https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
Done